src/client/Microsoft.Identity.Client/MsalError.cs

```diff
@@ -162,6 +162,12 @@ public static class MsalError
         /// </summary>
         public const string InvalidAuthorityType = "invalid_authority_type";
 
+
+        /// <summary>
+        /// The client is unauthorized to access resource. The error is returned in case of MAM. 
```

Comment:
Please add a comment like

 /// <summary>
        ///The client is unauthorized to access resource
        /// <para>What happens?</para>
        /// ....
        /// <para>Mitigation</para>
        /// ...
        /// </summary>